### ISSUE TITLE
How to select pretrained G epoch when training the full model

### ISSUE THREAD

**Issue by OwalnutO** [ly015/intrinsic_flow#6]
Thanks for your brilliant code.

When I try to train a full model as you suggested:
"# full (need a pretrained pose transfer model without pixel warping)"

How do I select the pre-trained G epoch? I the REDME.md, you selected the 8-th epoch. 
"--pretrained_G_id id_pose_4 --pretrained_G_epoch 8"

But according to my results obtained after training the model without pixel warp, the best model is 5-th epoch, and the visualization results suggest that the results from last epoch are more better than previous models. So, with what metric do you select the pre-trained model?

**Comment by ly015** [comment]
Here we don't have a specific metric for selecting the pretrained model. Usually, we simply compare the visualized results after each epoch and choose the visually best one. We observed that as the training progresses, the L1 term and perceptual term will conquer the loss. So it may be helpful to track the gradient information and stop training when the gradient of the adversarial loss vanishes.
